raster/src/main/scala/geotrellis/raster/io/geotiff/tags/TiffTags.scala

```diff
@@ -181,16 +181,36 @@ case class TiffTags(
 
   def bandType: BandType = {
     val sampleFormat =
-      (this 
+      (this
         &|-> TiffTags._dataSampleFormatTags
         ^|-> DataSampleFormatTags._sampleFormat get)
 
     BandType(bitsPerSample, sampleFormat)
   }
 
-  def crs: CRS = proj4String match {
-    case Some(s) => CRS.fromString(s)
-    case None => LatLng
+  def proj4String: Option[String] = try {
+    GeoTiffCSTags.getProj4String
+  } catch {
+    case e: Exception => {
+      None
+    }
+  }
+
+  def pcs: Int = try {
```

Comment:
Ah, I see. Projected Coordinate System. I think this name deserves not being an initialism.